source/examples/demo-stages/source/ColorizeStage.cpp

```diff
@@ -1,5 +1,5 @@
 
-#include <gloperate/stages/demos/ColorizeStage.h>
+#include <demo-stages/ColorizeStage.h>
```

Comment:
As the headers and sources of the examples are to be placed flat in the demo-stages directory, the include paths are also flat. Further, you should use the ```#include "ColorizeStage.h"``` variant for non-library includes.